repo-add: add page (#6622)
tldr-pages/tldr: pages/linux/repo-add.md

# repo-add

> > Package database maintenance utility which enables installation via pacman.

Reviewer: Does this mean that:

1. It enables me to install a given package file with `pacman`
2. pacman can't work without it

Author: Number 1. I'll cleanup that part.

Author: https://github.com/tldr-pages/tldr/pull/6622/commits/3aadcab4afac46d4fed555d76dfcb65fa95db507 I hope the following commit helps.

> > Package database maintenance utility which enables installation of said package via pacman.

Reviewer 2: ```suggestion
> Package database maintenance utility which enables installation of said package via Pacman.
> More information: <https://man.archlinux.org/man/repo-add>.
```

> - Add all package binaries in the current directory and remove the old database file:

Reviewer: Add all package binaries in the current directory to where?

Author: To a database file in the same directory as shown in the following image. To put it up 1 directory, you'd run ../yourdatabasename.db.tar.gz
![image](https://user-images.githubusercontent.com/60808802/135693525-324c1643-91c4-46cd-8ad1-ed866320084e.png)

> `repo-add --remove {{your_database_name.db.tar.gz *.pkg.tar.zst}}`

Reviewer 3: ```suggestion
`repo-add --remove {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```

- Add all package binaries in the current directory in silent mode except for warning and error messages:

> `repo-add --quiet {{your_database_name.db.tar.gz *.pkg.tar.zst}}`

Reviewer 3: ```suggestion
`repo-add --quiet {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```

> - Remove color from output:
> 
> `repo-add --nocolor {{your_database_name.db.tar.gz *.pkg.tar.zst}}`

Reviewer 3: ```suggestion
- Add all package binaries in the current directory without showing color:

`repo-add --nocolor {{path/to/database.db.tar.gz}} {{*.pkg.tar.zst}}`
```
